apps/admin/src/app/workspace/workspace-layout-responsive.component.scss

```
@@ -0,0 +1,73 @@
+  .nav-label {
+    display: block;
+    max-width: 12rem;
+    opacity: 1;
+    visibility: visible;
+  }
+
+  .nav-item {
+    justify-content: flex-start;
+    gap: 0.75rem;
+
+    height: 48px;
```

Stylelint (17.14.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)